__tests__/lib/dev-environment/dev-environment-cli.js

```diff
@@ -150,14 +151,22 @@ describe( 'lib/dev-environment/dev-environment-cli', () => {
 					tag: '/tmp/wp',
 				},
 			},
-			{ // if local is  allowed
+			// if local is allowed
+			{ ( os.platform() == 'win32' ? { 
+				param: 'C:\path',
+				allowLocal: true,
+				expected: {
+					mode: 'local',
+					dir: 'C:\path',
```

Comment:
```suggestion
				param: 'C:\\path',
				allowLocal: true,
				expected: {
					mode: 'local',
					dir: 'C:\\path',
```

because

```
console.log('C:\path');
C:path
```